src/test/kotlin/com/terraformation/backend/customer/model/PermissionTest.kt

```diff
@@ -2602,6 +2618,17 @@ internal class PermissionTest : DatabaseTest() {
       }
     }
 
+    fun expect(
```

Comment:
Need to call this in `andNothingElse` too so it tests that the permission is false for regular users.